eslint.config.mjs

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import prettierConfig from 'eslint-config-prettier';
+import config from '@zendeskgarden/eslint-config';
+import noticePlugin from '@zendeskgarden/eslint-config/plugins/notice.js';
+import reactPlugin from '@zendeskgarden/eslint-config/plugins/react.js';
+import typescriptPlugin from '@zendeskgarden/eslint-config/plugins/typescript.js';
+import jestPlugin from '@zendeskgarden/eslint-config/plugins/jest.js';
+
+const typescriptRules = {
+  ...typescriptPlugin.rules,
+  '@typescript-eslint/explicit-function-return-type': 'off',
+  '@typescript-eslint/explicit-module-boundary-types': 'off',
+  '@typescript-eslint/naming-convention': 'off',
+  '@typescript-eslint/no-explicit-any': 'off',
+  'n/no-unsupported-features/es-builtins': ['error', { version: '>=16.9.0' }],
+  'react/prop-types': 'off',
+  'react/jsx-no-leaked-render': 'off'
```

Comment:
Rule recommended to be turned off in codebases "... which encourage [s] you to always use boolean conditions." 

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-leaked-render.md#when-not-to-use-it

I couldn't find other eslint rules which enforce for booleans for render conditions, but i also know we're pretty good about avoiding these accidental render issues, so I turned it off.